lighty-core/lighty-controller-guice-di/src/test/java/io/lighty/core/controller/guice/tests/GuiceDITest.java

```diff
@@ -0,0 +1,228 @@
+    }
+
+    @Test
+    public void testDIClusterSingletonServiceProvider() {
+        Assert.assertNotNull(testService.getClusterSingletonServiceProvider());
+    }
+
+    @Test
+    public void testDIEventExecutor() {
+        Assert.assertNotNull(testService.getEventExecutor());
+    }
+
+    @Test
+    public void testDIEventLoopGroupBoss() {
+        Assert.assertNotNull(testService.getEventLoopGroupBoss());
+    }
+
+    @Test
+    public void testDIEventLoopGroupWorker() {
+        Assert.assertNotNull(testService.getEventLoopGroupWorker());
+    }
+
+    @Test
+    public void testDIThreadPool() {
+        Assert.assertNotNull(testService.getThreadPool());
+    }
+
+    @Test
+    public void testDIScheduledThreadPool() {
+        Assert.assertNotNull(testService.getScheduledThreadPool());
+    }
+
+    @Test
+    public void testDITimer() {
+        Assert.assertNotNull(testService.getTimer());
+    }
+
+    @Test
+    public void testDIDOMMountPointService() {
+        Assert.assertNotNull(testService.getDomMountPointService());
+    }
+
+    @Test
+    public void testDIDOMNotificationPublishService() {
+        Assert.assertNotNull(testService.getDomNotificationPublishService());
+    }
+
+    @Test
+    public void testDIDOMNotificationService() {
+        Assert.assertNotNull(testService.getDomNotificationService());
+    }
+
+    @Test
+    public void testDIDOMDataBroker() {
+        Assert.assertNotNull(testService.getDomDataBroker());
+    }
+
+    @Test
+    public void testDIDOMRpcService() {
+        Assert.assertNotNull(testService.getDomRpcService());
+    }
+
+    @Test
+    public void testDIDOMRpcProviderService() {
+        Assert.assertNotNull(testService.getDomRpcProviderService());
+    }
+
+    @Test
+    public void testDIRpcProviderService() {
+        Assert.assertNotNull(testService.getRpcProviderService());
+    }
+
+    @Test
+    public void testDIMountPointService() {
+        Assert.assertNotNull(testService.getMountPointService());
+    }
+
+    @Test
+    public void testDINotificationService() {
+        Assert.assertNotNull(testService.getNotificationService());
+    }
+
+    @Test
+    public void testDINotificationPublishService() {
+        Assert.assertNotNull(testService.getNotificationPublishService());
+    }
+
+    @Test
+    public void testDILightyServices() {
+        Assert.assertNotNull(testService.getLightyServices());
+    }
+
+    @Test
+    public void testDILightyModuleRegistryService() {
+        Assert.assertNotNull(testService.getLightyModuleRegistryService());
+    }
+
+    @Test
+    public void testDIBindingDataBroker() {
+        Assert.assertNotNull(testService.getBindingDataBroker());
+    }
+
+    @AfterClass
+    public static void shutdown() {
+        try {
+            if (lightyController != null) {
+                lightyController.shutdown();
+            }
+        } catch (Exception e) {
+            Assert.fail();
```

Comment:
this should just throw, not mask the failure behind an Assert.fail() -- which test is it failing really?!

Reply:
Sonar does not like "throws Exception" on method level, nor empty catch statement. that's why Assert.fail(); inside catch.